lib/matplotlib/backends/backend_gtk3.py

```diff
@@ -761,13 +834,12 @@ def draw_rubberband(self, x0, y0, x1, y1):
 
 
 class ToolbarGTK3(ToolContainerBase, Gtk.Box):
-    def __init__(self, toolmanager):
+    def __init__(self, toolmanager, flow='horizontal'):
```

Comment:
In the same way that we handle ``ToolbarGTK3.add_tool``, i.e. by not handling it.  Why would the user try to set the ``flow`` property of a toolbar (or use ``add_tool``) if they hadn't set ``rcParams['toolbar'] = 'toolmanager'``?